console/metrics.adoc

```diff
@@ -0,0 +1,57 @@
+include::../partials/attributes.adoc[]
+[id="metrics"] 
+= Flow metrics
+
+This section shows the metrics available to a Prometheus instance when you enable the flow collector using `skupper init --enable-flow-collector`.
+Most metrics share a set of common labels as shown below, exceptions are shown beside appropriate metrics.
+
+NOTE: The metrics are available from `https://skupper.<namespace>.svc.clusterlocal:8010/api/v1alpha1/metrics`. Replace `<namespace>`` with the appropriate namespace where the {skupper-name} is deployed.
```

Comment:
Could not be worth mentioning in docs, but we do have a unresolved issue [SKUPPER-2077](https://issues.redhat.com/browse/SKUPPER-2077) around this: when --console-auth=openshift is set metrics are not available here.

In that case the port is 8443 instead of 8010, and is protected by the openshift oauth proxy. I personally haven't managed to sort out exactly how a service account/non-human could be able to authenticate to scrape these metrics, so do not have any useful guidance for this situation.

Reply:
@c-kruse  how about:

The metrics are available from  `https://skupper.<namespace>.svc.clusterlocal:8010/api/v1alpha1/metrics` when `console-auth` is set to 'internal' (default) or 'unsecured'.

and maybe we add SKUPPER-2077 as a known issue?

Reply:
That sounds great! Thanks!